app/controllers/hyrax/admin/workflow_roles_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Hyrax
+  module Admin
+    class WorkflowRolesController < ApplicationController
```

Comment:
Could you add to an initializer `WorkflowRolesController.before_action :index do ... end`?